### PR TITLE
Adding Documents to the master branch

### PR DESCRIPTION
**Pull request by shaivikantier**
The following readme files have been added to the master branch:
- MainRepo README
- xNFT README
- User Guide
- Single NFT Transfer guide
- Multi NFT Transfer guide

### CODE REVIEW

File: pallet-xnft/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# xNFT pallet
+
+Pallet for cross chain NFT transfer.
+
+## Overview
+
+The pallet contains the following features:-
+1. Transfers NFTs from parachain to parachain 
+2. Transfers collection from parachain to parachian
```
Comment (s-h-ubham) [inline pallet-xnft/README.md:9]: make this point 1

File: pallet-xnft/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+2. Transfers collection from parachain to parachian 
+
+#### **Please note:** In the latest version of the XCM protocol (v3), there's a limitation to the number of NFTs that can be transferred in a single transaction. Users are currently restricted to transferring a maximum of three NFTs at a time.. 
+
+## Installation
+
+#### Prerequisites:
+
+ - Rust installed (https://docs.substrate.io/install/)
+ -  Substrate version polkadot-v0.43 https://github.com/paritytech/substrate/tree/polkadot-v0.9.43)
```
Comment (s-h-ubham) [inline pallet-xnft/README.md]: remove extra space at start

File: pallet-xnft/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+3. Finally, connect the relay chain to your parachains to make a cross chain nft transfer. You can follow the below mentioned links for that:-
+- [Connect to a Relay Chain](https://docs.substrate.io/reference/how-to-guides/parachains/connect-to-a-relay-chain/)
+- [Connect to a Local Parachain](https://docs.substrate.io/tutorials/build-a-parachain/connect-a-local-parachain/)
+
+## Testing Guide
+- Run the following command to execute the test cases:
+```
+cargo test
```
Comment (s-h-ubham) [inline pallet-xnft/README.md]: replace this command with xNFT specific command

Comment (shaivikantier) [reply pallet-xnft/README.md]: This is related to a previous commit. Issue is already resolved with subsequent commits

File: pallet-xnft/README.md (hunk 1 above)
Comment (shaivikantier) [reply pallet-xnft/README.md:9]: Done

**Pull request closed without merging**